frontend/src/pages/Chat.jsx

```
@@ -0,0 +1,415 @@
+                          {/* Message content */}
+                          <div
+                            className={`
+                              py-2 px-4 rounded-2xl max-w-md w-fit break-words cursor-pointer shadow-sm
+                              ${isUserMessage
+                                ? "bg-gradient-to-r from-indigo-500 to-violet-600 text-white" 
+                                : "bg-white text-gray-800"}
+                            `}
+                            onClick={() => setTimestampMessageId(
+                              timestampMessageId === msg.id ? null : msg.id
+                            )}
+                          >
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)